### PR TITLE
Service feature profile

### PR DESCRIPTION
**Pull request by pbanascisco**
# Pull Request summary:
This PR adds all endpoints and models for Service Feature Profile (`/v1/feature-profile/sdwan/service`)

# Description of changes:
Endpoints added:
```
/v1/feature-profile/sdwan/service/{serviceId}/appqoe
/v1/feature-profile/sdwan/service/{serviceId}/dhcp-server
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn/{vpnId}/interface/ethernet
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn/{vpnId}/interface/gre
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn/{vpnId}/interface/ipsec
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn/{vpnId}/interface/svi
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn/{vpnId}/routing/bgp
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn/{vpnId}/routing/eigrp
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn/{vpnId}/routing/multicast
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn/{vpnId}/routing/ospf
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn/{vpnId}/routing/ospfv3/ipv4
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn/{vpnId}/routing/ospfv3/ipv6
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn/{vpnParcelId}/interface/ethernet/{ethernetId}/dhcp-server
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn/{vpnParcelId}/interface/ethernet/{ethernetId}/tracker
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn/{vpnParcelId}/interface/ethernet/{ethernetId}/trackergroup
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn/{vpnParcelId}/interface/ipsec/{ipsecId}/dhcp-server
/v1/feature-profile/sdwan/service/{serviceId}/lan/vpn/{vpnParcelId}/interface/svi/{sviId}/dhcp-server
/v1/feature-profile/sdwan/service/{serviceId}/routing/bgp
/v1/feature-profile/sdwan/service/{serviceId}/routing/eigrp
/v1/feature-profile/sdwan/service/{serviceId}/routing/multicast
/v1/feature-profile/sdwan/service/{serviceId}/routing/ospf
/v1/feature-profile/sdwan/service/{serviceId}/routing/ospfv3/ipv4
/v1/feature-profile/sdwan/service/{serviceId}/routing/ospfv3/ipv6
/v1/feature-profile/sdwan/service/{serviceId}/switchport
/v1/feature-profile/sdwan/service/{serviceId}/tracker
/v1/feature-profile/sdwan/service/{serviceId}/trackergroup
/v1/feature-profile/sdwan/service/{serviceId}/wirelesslan
/v1/feature-profile/sdwan/service/{service_id}/ipv6-acl
/v1/feature-profile/sdwan/service/{service_id}/objecttracker
/v1/feature-profile/sdwan/service/{service_id}/objecttrackergroup
/v1/feature-profile/sdwan/service/{service_id}/route-policy
/v1/feature-profile/sdwan/service/{service_id}/service-insertion-attachment
```

# Checklist:
- [x] Make sure to run pre-commit before committing changes
- [x] Make sure all checks have passed
- [x] PR description is clear and comprehensive
- [x] Mentioned the issue that this PR solves (if applicable)
- [x] Make sure you test the changes

### CODE REVIEW

**Review by igosoft — DISMISSED** [review]

File: vmngclient/model/configuration/feature_profile/sdwan/service/appqoe.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    model_config = ConfigDict(arbitrary_types_allowed=True, populate_by_name=True)
+
+    appnav_controller_group: List[ForwarderAndServiceNodeAppnavControllerGroup] = Field(alias="appnavControllerGroup")
+    service_node_group: List[ForwarderAndServiceNodeGroup] = Field(alias="serviceNodeGroup")
+
+    service_context: ServiceContext = Field(alias="serviceContext")
+
+
+# Service
+class ServiceNodeExternalAddress(str, Enum):
```
Comment (sbasan) [inline vmngclient/model/configuration/feature_profile/sdwan/service/appqoe.py:161]: This looks over complicated.
Why enum classes like `ServiceNodeExternalAddress`, `ServiceNodeExternalVpgIp` are needed? Looks like they store single default value.

File: endpoints-md.py [hunk 2]
```diff
@@ -104,6 +104,8 @@ def from_type_specifier(typespec: TypeSpecifier) -> CompositeTypeLink:
             if payloadtype := typespec.payload_type:
                 if payloadtype.__module__ == "builtins":
                     return CompositeTypeLink.text_only(payloadtype.__name__)
+                if issubclass(payloadtype, Generic):  # type: ignore
```
Comment (sbasan) [inline endpoints-md.py:107]: Can we remove that? I can't see the use case.
It also breaks previously working markdown links to `MigrationFile`, `SoftwarePackageUpdatePayload`...

**Review by sbasan — CHANGES_REQUESTED** [review]

File: vmngclient/model/configuration/feature_profile/sdwan/service/appqoe.py (hunk 1 above)
Comment (tehAgitto) [reply vmngclient/model/configuration/feature_profile/sdwan/service/appqoe.py:161]: Models will be rewritten. Merging before `catalystwan` name change.

**Review by tehAgitto — APPROVED** [review]

**Review by sbasan — APPROVED** [review]
lgtm

**Pull request merged**